### DISCUSSION TITLE
Python Project pyproject.toml config

### DISCUSSION THREAD

**Discussion by HarryAnkers** [projen/projen#2723] [Q&A]
Is it possible to add config for the dependancies itself to the pyproject.toml.
For instance I have some formatter configuration:
"""
[tool.black]
line-length = 120
target-version = ['py39']

[tool.isort]
profile = "black"
line_length = 120
extend_skip = ["__init__.py"]
"""
But I can't seem to find a projen command to add this to the .toml.

**Comment by HarryAnkers** [comment]
The closest I came up with was:
"""
# Change file permissions to allow write access
os.chmod("pyproject.toml", 0o777)  # Replace with the actual file path

# Open the pyproject.toml file in append mode
with open("pyproject.toml", "a") as file:
    # Write the [tool.black] section
    file.write("\n[tool.black]\n")
    file.write("line-length = 120\n")
    file.write("target-version = ['py37']\n")
    file.write("extend-exclude = '''\n")
    file.write("/(\n")
    file.write("    \\.git\n")
    file.write("  | \\.vscode\n")
    file.write("  | venv\n")
    file.write("  | __init__.py\n")
    file.write("  | \\.vscode\n")
    file.write(")/\n")
    file.write("'''\n\n")

    # Write the [tool.isort] section
    file.write("[tool.isort]\n")
    file.write('profile = "black"\n')
    file.write("line_length = 120\n")
    file.write('extend_skip = ["__init__.py"]\n')
"""

But this is obviously messy

**Comment by mrgrain** [comment]
Try this:
```python
project.try_find_object_file("pyproject.toml").add_override("tool.black", {
    "line-length": 120,
    "target-version": ["py39"]
})
```
